troubleshooting/troubleshooting-vse-path-too-long-exception.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Path Too Long Exception when Building Project in Microsoft Visual Studio
+description: AppBuilder Projects with fully qualified file names longer that 260 symbols fail to build in Visual Studio Extension.
+type: troubleshooting
+page_title: VS Extension Build Fails when File Path Length is Longer than 260 Symbols 
+slug: vse-path-too-long-exception
+position:
+tags: appbuilder, vs-extension, build
+teampulseid:
+ticketid:  1044168, 1001926
+pitsid:
+
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>	
+  </tr>
+  <tr>
+    <td>Code (AppBuilder) client</td>
+    <td>Extension for Visual Studio</td>	
+  </tr>
+</table>
+
+## Description
+When you attempt to build your AppBuilder project in Microsoft Visual Studio, the following exception occurs: `System.IO.PathTooLongException`
+
+## Error Message
+###`System.IO.PathTooLongException`
+`The specified path, file name, or both are too long. The fully qualified file name must be less than 260 characters, and the directory name must be less than 248 characters.`
+
+## Cause
+This issue is caused by a restriction in the Microsoft Windows API. More information about this is available here:
+
+* [Windows API maximum path length limitation](https://msdn.microsoft.com/en-us/library/windows/desktop/aa365247(v=vs.85).aspx#maxpath)
+
+## Solution
+Locate any fully qualified file names longer than 218 symbols. Achieve a path length that does not exceed the restricted value by modifying the file structure or renaming some of the files or folders.
```

Comment:
Consider including a link to "fully qualified file names".

Myself, I would probably rework the solution into something like "Verify that your project does not contain any fully qualified file names longer than 218 characters." And then follow it with the two simple steps of "1. Locate ..." and "2. Modify the file structure or rename..."